exchange/impcustomcachekeytest/multiImpBanners.json

```diff
@@ -0,0 +1,72 @@
+{
+	"bidRequest": {
+		"imp": [{
+				"id": "oneImp",
+				"exp": 300
+			},
+			{
+				"id": "twoImp"
+			}
+		]
+	},
+	"pbsBids": [{
+		"bid": {
+			"id": "bidOne",
+			"impid": "oneImp",
+			"price": 7.64,
+			"exp": 600,
+			"cat": ["11_sports_22"]
+		},
+		"bidType": "banner",
+		"bidder": "appnexus"
+	}, {
+		"bid": {
+			"id": "bidTwo",
+			"impid": "oneImp",
+			"price": 15.64,
+			"exp": 200,
+			"cat": ["33_news_44"]
+		},
+		"bidType": "banner",
+		"bidder": "pubmatic"
+	}, {
+		"bid": {
+			"id": "bidThree",
+			"impid": "oneImp",
+			"price": 2.3,
+			"cat": ["33_news_44"]
```

Comment:
Another minor nit probably not requiring a change:  logic in PBS will prevent two bids for the same imp from having the same cat value so the scenario here could never actually happen.  Since cat will not be used because none of these bids will get cached anyway, it's probably not worth worrying about.

Reply:
it's not a big change and I could still make it. What do you think?

Reply:
Unless someone else objects, I'm okay with the code as is.